### ISSUE TITLE
annotation problem

### ISSUE THREAD

**Issue by woodoo46** [PapenfussLab/gridss#296]
We are annotating calls on mouse genome mm10. We had this problem:

_Warning: 442425 parsing failures.
    row col expected     actual                      file
4344472  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344473  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344474  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344475  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344476  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
....... ... ........ .......... .........................
See problems(...) for more details._

I am guessing the chromosome names may be a problem, we had chromosome names 1, 2 3, ..., X and Y etc. 

Thanks!

**Comment by d-cameron** [comment]
Are you able to supply the log file? It's not entirely clear which step this is failing on.

**Comment by woodoo46** [comment]
1. I ran gridss.sh supplied in the pacakge generating a vcf file, this finished successfully. We are analyzing mouse data.
2. I tried to annotate the SV types by using this command (may I have missed something in my command line):

Rscript ${GRIDSS_PATH}/gridss_annotate_insertions_repeatmaster.R \
        --input ${inputfile} \
        --output ${outputfile}\
        --repeatmasker ${genome}.out \
        --scriptdir ${GRIDSS_PATH}

I had the following errors:
Warning: 442425 parsing failures.
    row col expected     actual                      file
4344472  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344473  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344474  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344475  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
4344476  X5 a double GL456210.1 'ref/GRCm38_68.fasta.out'
....... ... ........ .......... .........................
See problems(...) for more details.

Error in .normarg_seqnames1(seqnames) : 'seqnames' cannot contain NAs
Calls: import.repeatmasker.fa.out -> GRanges -> new_GRanges -> .normarg_seqnames1
Execution halted

I am guessing something wrong with my chromosome names? I have 1, 2, ... as chromosome names. 

Thanks!
